factory/ethElrondBridgeComponents_test.go

```diff
@@ -125,6 +119,15 @@ func TestNewEthElrondBridgeComponents(t *testing.T) {
 		assert.Equal(t, errNilStatusStorer, err)
 		assert.Nil(t, components)
 	})
+	t.Run("nil MetricsHolder", func(t *testing.T) {
```

Comment:
move this before last test ("should work") to mentain the the order as in checkArgsEthereumToElrondBridge.
Also there is not test for empty NetworkAddress(GeneralConfig.Elrond.NetworkAddress)

Reply:
moved the test & added check